tests/unit/test_args.py

```diff
@@ -758,6 +758,32 @@ def test_account_id_endpoint_mode_disabled_on_unsigned_request(self):
         config = self.call_get_client_args()['client_config']
         self.assertEqual(config.account_id_endpoint_mode, 'disabled')
 
+    def test_disable_host_prefix_injection_default_client_config(self):
```

Comment:
Should this be `test_inject_host_prefix_default_client_config`?

Reply:
Fixed in fadc0255b080257fe358345bd2674bb18a75cebf